src/Containers/AllProducts.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import ProductComponent from './ProductComponents';
+import { myProducts } from '../Redux/Actions/productActions';
+
+const AllProducts = () => {
+  const dispatch = useDispatch();
+
+  const fetchProducts = async () => {
+    try {
+      const response = await axios.get('https://fakestoreapi.com/products');
+      dispatch(myProducts(response.data));
+    } catch (error) {
+      throw new Error('Error fetching products:', error);
+    }
+  };
+
+  useEffect(() => {
+    fetchProducts();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div>
+      <ProductComponent />
+    </div>
+  );
+};
+
```

Comment:
- [ ] Kindly ensure that the cards follow the alternating color pattern on the homepage similar to the original design, as indicated in the previous review. Difference between background color and each card the cards opposite on horizontal and vertical sides should be lighter and darker theme of the chosen color, like seen in the original design.


**Your design:**        |  **The origin design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Your desgin" src="https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/12745474/a4301a70-0a60-4209-bb23-f9310bc21872"> |  <img width="800" alt="Home Page" src="https://github.com/Dorcas126/capstone-project-metrics-webapp/assets/12745474/e91e9d83-4842-4934-b961-044e3dab00ed">